topics/bridgetownrb/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: bridgetownrb
+created_by: Jared White
+display_name: Bridgetown
+github_url: https://github.com/bridgetown
+logo: bridgetown.png
```

Comment:
According to the test failure, this needs to be the same name as the topic.

Reply:
Got it sorted out now.